client.go

```diff
@@ -102,13 +108,22 @@ func (c *Client) FetchProjectQuota(ctx context.Context, projectID uint64, now ti
 
 // FetchKeyQuota fetches and validates the accessKey from cache or from the quota server.
 func (c *Client) FetchKeyQuota(ctx context.Context, accessKey, origin string, now time.Time) (*proto.AccessQuota, error) {
+	logger := c.logger.With(
+		slog.String("op", "fetch_key_quota"),
+		slog.String("access_key", accessKey),
+	)
 	// fetch access quota
 	quota, err := c.quotaCache.GetAccessQuota(ctx, accessKey)
 	if err != nil {
 		if !errors.Is(err, proto.ErrAccessKeyNotFound) {
+			logger.Warn("unexpected cache error", slog.Any("err", err))
 			return nil, err
 		}
 		if quota, err = c.quotaClient.GetAccessQuota(ctx, accessKey, now); err != nil {
+			if !errors.Is(err, proto.ErrAccessKeyNotFound) {
```

Comment:
This is the only error we want, anything else is some unexpected error. We want the requests to go through in that case.